internal/app/frontend/frontend_service_test.go

```diff
@@ -79,6 +81,83 @@ func TestDoCreateTickets(t *testing.T) {
 	}
 }
 
+func TestDoGetAssignments(t *testing.T) {
+	testTicket := &pb.Ticket{
+		Id: "test-id",
+	}
+
+	totalAssignments := 2
+	var wantAssignments []*pb.Assignment
+
+	for i := 0; i < totalAssignments; i++ {
+		wantAssignments = append(wantAssignments, &pb.Assignment{Connection: string(i)})
```

Comment:
This should be a part of the test cases. So wantAssignments []*pb.Assignment in the struct. 

In the first test case, since you expect error - it will be empty / uninitialized. 
In the second one, you can initialize it. Also, since these are static expected values, you could simply then initialize as []*pb.Assignment{ <actually just have the assignment objects>}

Thus everythign specific to the test case is within the struct.